src/surface_conditions/surface_conditions.jl

```diff
@@ -335,21 +329,51 @@ function surface_state_to_conditions(
 end
 
 #Sphere SST distribution from Wing et al. (2023) https://gmd.copernicus.org/preprints/gmd-2023-235/
-function surface_temperature(::RCEMIPIISphereSST, coordinates)
+function surface_temperature(
+    ::RCEMIPIISST,
+    coordinates::Union{Geometry.LatLongZPoint, Geometry.LatLongPoint},
+)
     (; lat) = coordinates
     FT = eltype(lat)
     T = FT(300) + FT(1.25) / 2 * cosd(360 * lat / 54)
     return T
 end
 
-function surface_temperature(::ZonallySymmetricSST, coordinates)
+#Plane SST distribution from Wing et al. (2023) https://gmd.copernicus.org/preprints/gmd-2023-235/
+function surface_temperature(
+    ::RCEMIPIISST,
+    coordinates::Union{Geometry.XYPoint, Geometry.XYZPoint},
+)
+    (; x) = coordinates
+    FT = eltype(x)
+    T = FT(300) + FT(1.25) / 2 * cos(2 * FT(pi) * x / 6000)
+    return T
+end
+
+#For non-RCEMIPII box models with prescribed surface temp, assume that the latitude is 0.
+function surface_temperature(
+    ::Union{ZonallySymmetricSST, ZonallyAsymmetricSST},
+    coordinates::Union{Geometry.XYPoint, Geometry.XYZPoint},
```

Comment:
This also needs to work with `XZPoint` to support the "2D" extruded configuration.